src/convertToObject.js

```diff
@@ -11,8 +11,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = {};
+
+  sourceString
+    .split(';')
+    .map(stile => stile.trim())
+    .filter(style => style.length > 0)
```

Comment:
you can remove this line and check if elements exist before adding them to an object